pkgs/development/libraries/liberfa/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "erfa";
+  version = "1.7.0";
+
+  buildInputs = [ autoreconfHook ];
+
+  src = fetchFromGitHub {
+    owner = "liberfa";
+    repo = "erfa";
+    rev = "v${version}";
+    sha256 = "1z4k2phrw6wwi0kax6ac80jk9c036gi7pmhmg6gaf3lk81k6xz2r";
+  };
+
+  configureFlags = [ "--enable-shared" ];
+
+  meta = with stdenv.lib; {
+    description = "A C library containing key algorithms for astronomy, and is based on the SOFA library published by the International Astronomical Union (IAU).";
```

Comment:
This should not end on a period: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

Also its a bit long. Why not reuse the more succinct upstream tagline "Essential Routines for Fundamental Astronomy"?

Reply:
All right, I'll put the short description.